packages/@ama-sdk/core/src/utils/json-token.ts

```diff
@@ -148,8 +148,8 @@ export function createJweEncoder(aesTagLengthInBits = 128, useHeaderAsAAD = fals
   const base64Encoder = createBase64UrlEncoder();
   const stringEncoder = typeof window.TextEncoder !== 'undefined' ? new TextEncoder() : new Encoder();
 
-  return async ({publicKey, keyId}: {publicKey: Promise<CryptoKey> | CryptoKey; keyId: string}, jwePayload: Record<string, unknown>, publicProperties: string[]) => {
-    const jweHeader = {
+  return async ({ publicKey, keyId }: { publicKey: Promise<CryptoKey> | CryptoKey; keyId: string }, jwePayload: Record<string, unknown>, publicProperties: string[]) => {
+    const jweHeader: Record<string, unknown> = {
```

Comment:
why `unknown`?

Reply:
Coming from `jwePayload` type, I kept the original type